h2spec.go

```diff
@@ -248,7 +262,7 @@ func (tc *TestCase) PrintEphemeralDesc() {
 func (tc *TestCase) PrintPass() {
 	mark := "✓"
 	logger.Clear()
-	logger.Write("\x1b[32m%s\x1b[0m \x1b[90m%s\x1b[0m\n", mark, tc.Desc)
+	logger.Write("\x1b[32m%s\x1b[0m \x1b[90m%s; test time: %s\x1b[0m\n", mark, tc.Desc, tc.testTime)
```

Comment:
Personally, I think that it's unnecessary because this is complicated to show the description.